apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page.module.ts

```diff
@@ -0,0 +1,50 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
```

Comment:
imports arrangement external, internal and local files, For more details angular guidelines will help.

Reply:
fixed for this file(was already arranged for other files) and added spacings between the imports.